src/SprykerEco/Zed/Adyen/Business/Handler/Notification/AdyenNotificationHandler.php

```diff
@@ -61,38 +61,30 @@ public function __construct(
      */
     public function handle(AdyenNotificationsTransfer $notificationsTransfer): AdyenNotificationsTransfer
     {
-        $isSuccess = true;
+        $this->writer->saveNotifications($notificationsTransfer);
 
         foreach ($notificationsTransfer->getNotificationItems() as $notificationItem) {
-            if ($this->handleNotification($notificationItem) === false) {
-                $isSuccess = false;
-            }
+            $this->handleNotification($notificationItem);
         }
 
-        $notificationsTransfer->setIsSuccess($isSuccess);
-
         return $notificationsTransfer;
     }
 
     /**
      * @param \Generated\Shared\Transfer\AdyenNotificationRequestItemTransfer $notificationTransfer
      *
-     * @return bool
+     * @return void
      */
-    protected function handleNotification(AdyenNotificationRequestItemTransfer $notificationTransfer): bool
+    protected function handleNotification(AdyenNotificationRequestItemTransfer $notificationTransfer): void
     {
-        if (!$notificationTransfer->getSuccess()) {
-            return false;
-        }
-
         $statuses = $this->config->getMappedOmsStatuses();
         if (!array_key_exists($notificationTransfer->getEventCode(), $statuses)) {
```

Comment:
Please check that the `$notificationTransfer->getSuccess()` key also exists. On the project level someone might not add one and you get fatal error below in line 97.